homeassistant/components/futurenow/light.py

```diff
@@ -51,7 +51,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 
 def to_futurenow_level(level):
     """Convert the given Home Assistant light level (0-255) to FutureNow (0-100)."""
-    return int((level * 100) / 255)
+    return int((level * 100) / 255 + 0.5)
```

Comment:
I think you are looking for rounding?

```suggestion
    return round((level * 100) / 255)
```

Reply:
Right! Thanks for the suggestion.

I'm more used to Lua where there is no built-in round function.

Reply:
@spike411 you should probably commit the suggestion in order to get this PR moving 😉